Gemfile

```diff
@@ -1,8 +1,7 @@
 source 'https://rubygems.org'
 
-
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.7.1'
+gem 'rails', '5.0'
```

Comment:
This seems a bit too strict.  I think if you use something along the lines of `~> '5.0.0'` you'll be able to upgrade to different patch versions of rails (for instance, 5.0.2 is what we should probably be using, but the way this is specified we're only going to get 5.0.0 exactly).

Luckily you've done the heavy lifting updating to rails 5, so that's a simple change.